index.html

```diff
@@ -37,6 +37,128 @@ <h1 class="name">
 			<div class="bottom_geometry"></div>
 		</section>
 
+		<section class="project_section">
+			<h1 class="projects_label">
+				Projects
+			</h1>
+			<div class="project_card">
+				<div class="project_image"></div>
+				<div class="project_details_card">
+					<p class="project_name">
+						Bavon's Portfolio
+					</p>
+					<div class="project_tecs_container">
+						<div class="tech_card">
+							<p class="tech_text">
+								HTML/CSS
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								Ruby on Rails
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								JavaScript
+							</p>
+						</div>
+					</div>
+					<div class="see_project">
+						<p class="see_project_text">
+							See this project
+						</p>
+						<div class="arrow_forward"></div>
+					</div>
+				</div>
+			</div>
+			<div class="project_card">
+				<div class="project_image_1"></div>
+				<div class="project_details_card">
+					<p class="project_name">
+						Bavon's Portfolio
+					</p>
+					<div class="project_tecs_container">
+						<div class="tech_card">
+							<p class="tech_text">
+								HTML/CSS
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								Ruby on Rails
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								JavaScript
+							</p>
+						</div>
+					</div>
+					<div class="see_project">
+						<p class="see_project_text">
+							See this project
+						</p>
+						<div class="arrow_forward"></div>
+					</div>
+				</div>
+			</div>
+		</section>
+		<section class="about_section">
+			<h1 class="about_title">
+				About
+			</h1>
+
+			<p class="about_paragraph">
+				Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+				dolore magna
+				aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+			</p>
+
+			<div class="resume_button">
+				<p class="get_cv_text">
+					Get my CV
+				</p>
+				<div class="download_icon"></div>
+			</div>
+
+			<div class="my_langauges_card">
+				<h1 class="langauge_label">
+					Languages
+				</h1>
```

Comment:
- [ ] You have already one h1 tag on the page. The page should include only one [h1 tag](https://www.w3schools.com/tags/tag_hn.asp#:~:text=Note%3A%20Only%20use%20one%20%3Ch1%3E%20per%20page%20%2D%20this%20should%20represent%20the%20main%20heading/subject%20for%20the%20whole%20page.%20Also%2C%20do%20not%20skip%20heading%20levels%20%2D%20start%20with%20%3Ch1%3E%2C%20then%20use%20%3Ch2%3E%2C%20and%20so%20on.). Please change to another hx tag.